scripts/helper/create-role-bindings.sh

```diff
@@ -77,6 +78,24 @@ do
         --kafka-cluster-id $KAFKA_CLUSTER_ID
 done
 
+for resource in Topic:_confluent-license
```

Comment:
@sdanduConf with only one resource, can we remove the for loop?

e.g.

```
 confluent iam rolebinding create \
         --principal $SR_PRINCIPAL \
         --role DeveloperRead \
         --resource Topic:_confluent-license \
         --kafka-cluster-id $KAFKA_CLUSTER_ID
```

or

make the for loop to cover both roles.

Comment here applies to both SR and REST Proxy

Reply:
@ybyzek updated